org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterColorTemperature.java

```diff
@@ -62,24 +62,7 @@ public boolean initializeDevice() {
             return false;
         }
 
-        Integer minTemperatureInMired = serverClusterColorControl.getColorTemperatureMin(Long.MAX_VALUE);
-        Integer maxTemperatureInMired = serverClusterColorControl.getColorTemperatureMax(Long.MAX_VALUE);
-
-        // High Mired values correspond to low Kelvin values, hence the max Mired value yields the min Kelvin value
-        if (maxTemperatureInMired == null) {
-            kelvinMin = DEFAULT_MIN_TEMPERATURE_IN_KELVIN;
-        } else {
-            kelvinMin = miredToKelvin(maxTemperatureInMired);
-        }
-
-        // Low Mired values correspond to high Kelvin values, hence the min Mired value yields the max Kelvin value
-        if (minTemperatureInMired == null) {
-            kelvinMax = DEFAULT_MAX_TEMPERATURE_IN_KELVIN;
-        } else {
-            kelvinMax = miredToKelvin(minTemperatureInMired);
-        }
-
-        kelvinRange = kelvinMax - kelvinMin;
+        determineMinMaxTemperature(serverClusterColorControl);
```

Comment:
Why is this needed here? kelvinRange is not used in this method.

Reply:
Will remove it.